xbmc/windowing/XBMC_events.h

```diff
@@ -78,6 +80,10 @@ typedef struct XBMC_MoveEvent {
 	int y;		/* New y position */
 } XBMC_MoveEvent;
 
+struct XBMC_ModeChangeEvent {
```

Comment:
you are not the maintainer of this file. if you want to reformat it, you should to submit a separate pr for it. cosmetic changes just inflate the diff